docs/run-preinstalled-wda.md

```diff
@@ -80,3 +81,32 @@ Some 3rd party tools such as [ios-deploy](https://github.com/ios-control/ios-dep
 
 `WebDriverAgentRunner-Runner.app` package may exist in a `derivedDataPath` directory as explained in [Real Device Configuration tutorial](./real-device-config.md).
 The `WebDriverAgentRunner-Runner.app` can be installed without xcodebuild with the 3rd party tools.
+
+
+### Set `appium:prebuiltWDAPath`
+
+If `appium:prebuiltWDAPath` is provided with properly signed `WebDriverAgentRunner-Runner.app` test bundle (please check [Real Device Configuration tutorial](real-device-config.md)), XCUITest driver will install the application and launch it every session.
+The test bundle cannot set the versioning as `CFBundleVersion`, thus the installation occurs every session.
+
+The `.app` test bundle is generally generated as below if you built with Xcode.
```

Comment:
sorry, I don't understand what this sentence describes

Reply:
I tried to leave the default place for `WebDriverAgentRunner-Runner.app` when a user builds WDA via Xcode. Mostly for beginners.

Advanced users who can build with xcodebuild it not this scope.